
<%
    try {
        throw new Exception();
    } catch (Exception e) {
    	out.println("Exception stacktrace:");
        e.printStackTrace();
    }
%>
